datadog_lambda/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "5.92.0"
```

Comment:
Now we have to update this every time we do a release, could we add this to the internal docs?